README.md

```diff
@@ -53,13 +53,52 @@ $ git clone git@github.com:LBHackney-IT/social-care-case-viewer-api.git
 
 ### Running the application
 
-To serve the API locally, use:
+There are two ways of running the application: using dotnet or using docker.
+
+#### Using dotnet run
+Using the dotnet command will not automatically connect the API to any local database instances.
+
+To serve the API locally with dotnet,
+run `dotnet run` from within the [SocialCareCaseViewerApi](./SocialCareCaseViewerApi) project directory, i.e:
 
 ```sh
 $ cd SocialCareCaseViewerApi && dotnet run
 ```
 
-The application will be served at http://localhost:5000.
+**The application will be served at http://localhost:5000**.
+
+#### Using docker
+
+You can also run the API locally and have it connect to local databases running within docker.
+First make sure the local dev databases are running in docker:
+
+```sh
+$ make start-local-dev-dbs
+```
+
+Once these are running, you can run this command to build and run the API within the docker:
+
+```sh
+$ make serve
+```
+**The application will be served at http://localhost:3000**.
+
+N.B: This would only spin up the Application, Postgres & MongoDB locally in docker.
+It doesn't include setup for spinning up other APIs that this service connects to in Staging or in Production.
+
+The docker implementation is still a work-in-progress but the above setup should allow for some API calls to be tested locally.
+
+#### Troubleshooting
+
+While running the application and test databases using docker, when you attempt to test an API call, you may get a `500` error.
+If you see in the stacktrace an error when code in [SccvDbContext](./SocialCareCaseViewerApi/V1/Infrastructure/SccvDbContext.cs) was run, e.g:
+
+```sh
+System.ArgumentNullException: String reference not set to an instance of a String. (Parameter 's')
+```
+
+A quick fix for this to comment out the code in in [SccvDbContext](./SocialCareCaseViewerApi/V1/Infrastructure/SccvDbContext.cs) that sets up using a `localTrustStore` and adding a SSL certificate (e.g `lines 14 - 36`).
+After commenting this out, rerun `make serve` and try again.
```

Comment:
This isn't ideal - a potential workaround which I've tested is to use ASP.NET environment environment variable and wrap this in an `if` statement so:

- Add `ASPNETCORE_ENVIRONMENT=Development` to the environment for the `social-care-case-viewer-api:`
- Wrap lines 14 - 36 with an `if` statement with the condition `Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") != "Development"`

What do you think?

Reply:
Nice! I'll give it a try. I think quite a few of us have to comment this out to get it to work 😅

Reply:
Done in 21c72d7